src/components/Display/index.jsx

```diff
@@ -8,7 +8,7 @@ const Display = ({ number }) => (
 );
 
 Display.propTypes = {
-    number: PropTypes.number.isRequired,
+    number: PropTypes.string.isRequired,
```

Comment:
as a doubt, why you changed this to string?

Reply:
Because I will do a concatenation when you clic on a number button.

You clic 2, you have 2
You clic 3, you have 23